src/js/controller/instream-adapter.js

```diff
@@ -3,6 +3,7 @@ import { STATE_BUFFERING, STATE_COMPLETE, STATE_PAUSED,
     ERROR, MEDIA_META, MEDIA_TIME, MEDIA_COMPLETE,
     PLAYLIST_ITEM, PLAYLIST_COMPLETE, INSTREAM_CLICK, AD_SKIPPED } from 'events/events';
 import utils from 'utils/helpers';
+import { offsetSeconds } from 'utils/strings';
```

Comment:
`offsetSeconds` isn't a name which describes what the function does. What about something like `toSeconds`?

Reply:
@johnBartos  agreed, its not the best name but we already have a `seconds` function. The added benefit of this function is that it handles offsets that are percentages as well (i.e. '30%' ) . So this is specific to offsets. `offsetToSeconds`?